packages/gui/src/electron/main.tsx

```diff
@@ -321,6 +329,7 @@ if (!handleSquirrelEvent()) {
           nodeIntegration: true,
           contextIsolation: false,
           nativeWindowOpen: true,
+          webSecurity: false,
```

Comment:
Why you are disable webSecurity. If you have problem with CORS we can do it without disable of webSecurity

Reply:
We can re-enable this. For reading from the FS, we can have the renderer process communicate with the main process via ipcRenderer. See `getRemoteFileContent`